src/modules/75-cf/pages/feature-flags/components/FlagTableFilters.tsx

```diff
@@ -67,17 +67,31 @@ export const featureFlagFilters = (features: Features | null): Array<FilterProps
     total: features?.featureCounts?.totalPotentiallyStale || 0,
     tooltipId: 'ff_flagFilters_potentiallyStaleFlags',
     filterTotalColor: Color.ORANGE_800
+  },
+  {
+    queryProps: { key: FlagFilterKeys.ARCHIVED, value: 'true' },
+    label: 'cf.flagFilters.archivedFlags',
+    total: features?.featureCounts?.totalArchived || 0,
+    filterTotalColor: Color.GREY_700
   }
 ]
 
-export const FlagTableFilters: React.FC<FlagTableFiltersProps> = ({ features, currentFilter, updateTableFilter }) => {
-  const DISPLAY_ACTIVE_FILTER = useFeatureFlag(FeatureFlag.FFM_3938_STALE_FLAGS_ACTIVE_CARD_HIDE_SHOW)
+export const FlagTableFilters: FC<FlagTableFiltersProps> = ({ features, currentFilter, updateTableFilter }) => {
+  const {
+    FFM_3938_STALE_FLAGS_ACTIVE_CARD_HIDE_SHOW: DISPLAY_ACTIVE_FILTER,
+    FFM_7921_ARCHIVING_FEATURE_FLAGS: ARCHIVING_FEATURE_FLAGS
+  } = useFeatureFlags()
   let filters = featureFlagFilters(features)
 
   // remove 'Active Flags' filter card if feature flag FFM_3938_STALE_FLAGS_ACTIVE_CARD_HIDE_SHOW is not enabled
   if (!DISPLAY_ACTIVE_FILTER) {
     filters = filters.filter(filter => filter.label !== 'cf.flagFilters.active')
   }
 
+  // remove Archived filter card if there are no flags that are archived and if Archiving FF is toggled OFF
```

Comment:
Super minor, totally not a blocker
```suggestion
  // remove Archived filter card if there are no flags that are archived or if Archiving FF is toggled OFF
```